docs/user-guide/deploying-application/triggering-cd.md

```diff
@@ -1,70 +1,105 @@
 
-Select an image to deploy and then click `Deploy` to trigger the CD pipeline.
+    ![Figure 1: 'Select Image' Button](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/select-image.jpg)
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/deploy.jpg)
+2. Select an image to deploy and then click **Deploy** to trigger the CD pipeline.
 
-The currently deployed images are tagged as `Active on <Environment name>`.
+    ![Figure 2: Selecting an Image for Deployment](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/deploy-v2.jpg)
+
+However, if an image is already deployed, you can identify it by the tag `Active on <Environment name>`.
+
+![Figure 3: Currently Deployed Image](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/active-tag.jpg)
 
 ## Manual Approval for Deployment
 
-When manual approval is enabled for the deployment pipeline, you are required to request image approval before each deployment. Alternatively, you can only deploy images that have already been approved once.
+When [manual approval is enabled](../creating-application/workflow/cd-pipeline.md#4-manual-approval-for-deployment) for the deployment pipeline configured in the workflow, you are expected to request for an image approval before each deployment. Alternatively, you can deploy images that have already been approved once.
+
+If no approved images are available or the current image is already deployed, you won't see any images for deployment when clicking **Select Image**.
+
+![Figure 3: No Approved Image](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/no-approved-image-v2.jpg)
+
+### Requesting for Image Approval
+
+Users need to have [Build & deploy permission](../user-guide/global-configurations/authorization/user-access.md#role-based-access-levels) or above (along with access to the environment and application) to request for an image approval.
+
+To request an image approval, follow these steps:
+
+1. Navigate to the `Build & Deploy` page, and click the **Approval for deployment** icon.
+
+    ![Figure 4: Approval Button](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/deployment-approval-button-v2.jpg)
+
+2. Click the **Request Approval** button present on the image for which you want to request an approval and click **Submit Request**.
+
+    ![Figure 5: Requesting Approval](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-cd/request-approval-v2.jpg)
+
+    In case you have configured [SES or SMTP on Devtron](../global-configurations/manage-notification.md#notification-configurations), you can directly choose the approver(s) from the list of approvers as shown below.
```

Comment:
Please mention that the users selected here will be sent an approval request via email. Although, any user with approver permission for app + env in question would be able to approve the image.